src/Nimbus/Bus.cs

```diff
@@ -116,7 +116,7 @@ public IDeadLetterQueues DeadLetterQueues
 
                 await _messagePumpsManager.Start(messagePumpTypes);
             }
-            catch (AggregateException aex)
+            catch (Exception aex)
```

Comment:
Picky, but aex made sense for an aggregate exception, ex makes more sense if we're catching a generic exception.

If you can make that tiny change I'll merge (WIll take a few days to release, my build server is out of commission)